modules/stream-shredder/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/shredder/stream/State.scala

```diff
@@ -21,15 +21,15 @@ case class State(total: Long,
                  bad: Long,
                  minCollector: Option[Instant],
                  maxCollector: Option[Instant],
-                 types: Set[SchemaKey],
+                 types: Set[Data.ShreddedType],
```

Comment:
Just pointing out that we'll need an Iglu Central release if we want to send this out. Perhaps just `shredding_complete`, but `load_succeeded` would also be great.

Reply:
Yep, we will need to add new field to `shredding_complete` schema. But I am not sure what you mean with `load_succeeded`. Will it be new schema for some new feature ?

Reply:
`load_succeeded` already exists (we use 2-0-0, since 2.1.0) and it has everything from `shredding_completed` and some more stuff. So if we update `shredding_completed` then it gets changed as well.